services/queue/src/api.js

```diff
@@ -142,19 +143,23 @@ builder.declare({
     'not specified the queue may provide a default value.',
   ].join('\n'),
 }, async function(req, res) {
-  // Load Task entity
-  let task = await this.Task.load({
-    taskId: req.params.taskId,
-  }, true);
+  const {taskId} = req.params;
+
+  // Load Task entity if it's absent from cache
+  let task;
+  if (this.LRUcache.has(taskId)) {
+    task = this.LRUcache.get(taskId);
+  } else {
+    task = await this.Task.load({taskId}, true);
+    this.LRUcache.set(taskId, task);
+  }
```

Comment:
I think we need this sorta thing in other places where we load tasks such as `claimWork`

Reply:
That's not a bad idea, but let's start small with the caching and see what the impact on our load is.

Reply:
I don't think much of our load is coming from calls to `task` itself? I don't mind releasing this as-is though.